packages/react-native-layout/src/components/spacing/Spacing.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { View } from 'react-native';
+
+import type { TestableComponent } from '../../types/generic-props';
+
+import type { SpacingProps } from './Spacing.props';
+
+export const Spacing: React.FC<SpacingProps & TestableComponent> = ({
+  height,
+  width,
+  flex,
+  testID,
+}) => {
+  return (
+    /* eslint-disable react-native/no-inline-styles */
```

Comment:
ah yes, the familiar sight of an es-lint project :D 
maybe disable in config?

Reply:
@FabianMeul lets prove to the haters that ESLinst is love <3